app/src/main/res/xml/remote_config_akademiaandroida2023.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": {
+    "versionNumber": "2",
+    "updateTime": "2023-03-27T11:59:17.233899Z",
+    "updateUser": {
+      "email": "akademiaandroida2023@safetyheads.com"
+    },
+    "updateOrigin": "CONSOLE",
+    "updateType": "INCREMENTAL_UPDATE"
+  },
+  "parameterGroups": {
+    "task/49": {
+      "description": "version_name + api_url",
+      "parameters": {
+        "api_url": {
+          "defaultValue": {
+            "value": "https://safetyheads.com/"
+          },
+          "description": "Adres URL do API",
+          "valueType": "STRING"
+        },
+        "version_name": {
+          "defaultValue": {
+            "value": "{aktualna wersja aplikacji}"
```

Comment:
Should the `value` contain `{aktualna wersja aplikacji}`  Not `1.0.0` for example?

Reply:
@jordanrafalm 
In the task description it was mistake it should be `versionCode` not `versionName`. I updated description. @Kuba-Nalepa have right for now it should be `1.0.0` set in Firebase remote config console.

BTW do we need this json file in the project?